[2025-03-13T04:07:01.025+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-13T04:07:01.067+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_ingestion_gcs_dag.bigquery_external_table manual__2025-03-13T04:00:50.830640+00:00 [queued]>
[2025-03-13T04:07:01.091+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_ingestion_gcs_dag.bigquery_external_table manual__2025-03-13T04:00:50.830640+00:00 [queued]>
[2025-03-13T04:07:01.093+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-03-13T04:07:01.109+0000] {taskinstance.py:2890} INFO - Executing <Task(BigQueryCreateExternalTableOperator): bigquery_external_table> on 2025-03-13 04:00:50.830640+00:00
[2025-03-13T04:07:01.118+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1329) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-13T04:07:01.120+0000] {standard_task_runner.py:72} INFO - Started process 1331 to run task
[2025-03-13T04:07:01.120+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_gcs_dag', 'bigquery_external_table', 'manual__2025-03-13T04:00:50.830640+00:00', '--job-id', '511', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs_dag.py', '--cfg-path', '/tmp/tmpkht4yku8']
[2025-03-13T04:07:01.122+0000] {standard_task_runner.py:105} INFO - Job 511: Subtask bigquery_external_table
[2025-03-13T04:07:01.177+0000] {task_command.py:467} INFO - Running <TaskInstance: data_ingestion_gcs_dag.bigquery_external_table manual__2025-03-13T04:00:50.830640+00:00 [running]> on host b0577ed9eb82
[2025-03-13T04:07:01.284+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_ingestion_gcs_dag' AIRFLOW_CTX_TASK_ID='bigquery_external_table' AIRFLOW_CTX_EXECUTION_DATE='2025-03-13T04:00:50.830640+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-13T04:00:50.830640+00:00'
[2025-03-13T04:07:01.287+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-03-13T04:07:01.287+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-03-13T04:07:01.288+0000] {logging_mixin.py:190} INFO - Current task name:bigquery_external_table state:running start_date:2025-03-13 04:07:01.068418+00:00
[2025-03-13T04:07:01.288+0000] {logging_mixin.py:190} INFO - Dag name:data_ingestion_gcs_dag and current dag run status:running
[2025-03-13T04:07:01.289+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-13T04:07:01.291+0000] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2025-03-13T04:07:01.292+0000] {connection.py:277} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2025-03-13T04:07:01.293+0000] {base.py:84} INFO - Retrieving connection 'google_cloud_default'
[2025-03-13T04:07:04.904+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-13T04:07:04.905+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=data_ingestion_gcs_dag, task_id=bigquery_external_table, run_id=manual__2025-03-13T04:00:50.830640+00:00, execution_date=20250313T040050, start_date=20250313T040701, end_date=20250313T040704
[2025-03-13T04:07:04.945+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-03-13T04:07:04.946+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-03-13T04:07:04.946+0000] {logging_mixin.py:190} INFO - Dag name:data_ingestion_gcs_dag queued_at:2025-03-13 04:00:50.872907+00:00
[2025-03-13T04:07:04.947+0000] {logging_mixin.py:190} INFO - Task hostname:b0577ed9eb82 operator:BigQueryCreateExternalTableOperator
[2025-03-13T04:07:04.992+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-13T04:07:05.017+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-13T04:07:05.020+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
